sympy/plotting/plot.py

```diff
@@ -2216,41 +2223,44 @@ def plot3d(*args, show=True, **kwargs):
     Arguments
     =========
 
-    ``expr`` : Expression representing the function along x.
+    expr : Expression representing the function along x.
 
-    ``range_x``: (x, 0, 5), A 3-tuple denoting the range of the x
-    variable.
+    range_x : (x, 0, 5), A 3-tuple denoting the range of the x
+    variable,
```

Comment:
on 2101 (in the new file) the tuple was placed at the end. I think it would be better to make this consistent. Why the change of `.` to `,` here?

Reply:
I moved it back and forgot to change back the , to .

Will think about how to write this. I am not really happy with any of the current approaches. (But want to see if the matplotlib is working.)

Reply:
I changed these to 

```
    range_x : (:class:`Symbol`, float, float)
        A 3-tuple denoting the range of the x variable, e.g. (x, 0, 5).
```